exdb/forms.py

```diff
@@ -90,6 +119,11 @@ def clean(self):
         if ex_type.needs_verification and self.cleaned_data.get('attendance'):
             raise ValidationError(_('%(name)s events cannot have an attendance') % {'name': ex_type.name})
 
+        # There are too many branches in this function; this is fixed on
+        # the approval validation branch.
+        if not ex_type.needs_verification and not self.cleaned_data.get('conclusion'):
+            raise ValidationError(_('%(name)s events must have a conclusion') % {'name': ex_type.name})
+
```

Comment:
You should check for people who pick a Type that needs verification and accidentally include a value for conclusion. Just set it to the empty string if that happens.